build.gradle

```diff
@@ -1,17 +1,19 @@
 plugins {
-	id "java"
-	id "java-library"
-	id "maven-publish"
-	id "net.minecrell.licenser" version "0.4.1"
+	id 'java-library'
+	id 'maven-publish'
+	id 'org.cadixdev.licenser'
+	id 'checkstyle'
 }
 
-sourceCompatibility = 1.8
+checkstyle {
+	configFile = file('checkstyle.xml')
+}
 
-def ENV = System.getenv()
-version = "0.6.2" + (ENV.GITHUB_ACTIONS ? "" : "+local")
+sourceCompatibility = JavaVersion.VERSION_1_8
+targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
I wish we can bump our toolchain's java version. It does not have to be compatible with an old Minecraft's java version.

Reply:
I don't have anything against it, we can gladly bump it to 11 or 17. Though, @modmuss50 What's your opinion?

Reply:
Leaving it for now, we can always bump it later.